src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -83,28 +83,31 @@ public static class Server {
      * General configuration that doesn't need to be synchronized but needs to be available before server startup
      */
     public static class Common {
+        @Deprecated(since = "1.19.2", forRemoval = true)
         public final BooleanValue cachePackAccess;
+        @Deprecated(since = "1.19.2", forRemoval = true)
         public final BooleanValue indexVanillaPackCachesOnThread;
+        @Deprecated(since = "1.19.2", forRemoval = true)
         public final BooleanValue indexModPackCachesOnThread;
 
         Common(ForgeConfigSpec.Builder builder) {
-            builder.comment("General configuration settings")
+            builder.comment("[DEPRECATED / NO EFFECT]:General configuration settings")
```

Comment:
```suggestion
            builder.comment("[DEPRECATED / NO EFFECT]: General configuration settings")
```
Spacing for readability.